Makefile

```diff
@@ -1,6 +1,12 @@
 FIO_VERSION ?= 3.1
 FIO_TAG = v${FIO_VERSION}
 DOCKER_ID ?= stackhpc
+NUM_CLIENTS ?= 16
```

Comment:
I'm a bit confused about commit b580c7. It says it adds templater, but it does a lot of reformatting as well? It would be easier to review if those bits were kept in separate commits.